### PR TITLE
Weekly report

### PR DESCRIPTION
**Pull request by seanjhardy**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**

Weekly report fully works with error handling at every stage

**Semantic versioning classification:**

- [x] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

File: commands/skilltree/setup.js [hunk 1]
```diff
@@ -1,155 +1,215 @@
...
     }
+    startSetup(message, scope);
   }
   catch (error) {
     // messages in the same channel saying your DMs are disabled
-    message.channel.send(`<@${message.member.id}> Your DMs are Disabled. Please enable them and try again`);
-    message.channel.send("**HOW TO ENABLE DMS** ```\n\n 1) Right-click server icon \n 2) Click on Privacy Settings \n 3) Toggle \"Allow direct messages from server members on\" \n 4) press Done```");
-    return;
+    const errorMsg = await message.channel.send(
+      "<@${message.member.id}> Your DMs are Disabled. Please enable them and try again" +
```
Comment (vlyr) [inline commands/skilltree/setup.js:36]: The ${} formatting syntax only works in strings when using the ` ` quotes.

**Review by vlyr — DISMISSED** [review]

File: commands/skilltree/setup.js (hunk 1 above)
Comment (seanjhardy) [reply commands/skilltree/setup.js:36]: Oh true, good spot

**Review by vlyr — DISMISSED** [review]

**Review by swatarianess — DISMISSED** [review]

**Review by vlyr — APPROVED** [review]

**Review by thewildofficial — DISMISSED** [review]
i hope all 38 files are working but i cant be arsed to test everything so we'll see it as we go

**Comment by vlyr** [comment]
Some pretty complex conflicts here, who's gonna be the one to resolve them? :eyes:

**Comment by vlyr** [comment]
Ballsy

**Review by vlyr — APPROVED** [review]

**Pull request merged**